piet-wgsl/Cargo.toml

```diff
@@ -7,8 +7,144 @@ edition = "2021"
 
 [dependencies]
 wgpu = "0.14"
+wasm-bindgen = "0.2"
 raw-window-handle = "0.5"
 futures-intrusive = "0.5.0"
 parking_lot = "0.12"
 bytemuck = { version = "1.12.1", features = ["derive"] }
 piet-scene = { path = "../piet-scene" }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
Why are all of these dependencies needed? I can't see where they're used

Reply:
Good catch. Looks like we don't need any of these on the main crate. They're only needed for the example.